### ISSUE TITLE
Sqaud notworking

### ISSUE THREAD

**Issue by ino89** [Austinb/GameQ#375]
Hello,

Sqaud ist notworking
server ist offline

**Comment by Austinb** [comment]
@ino89 What is the IP:port of the server you are trying to query?  Saying it is not working is not enough information as there could be many issues preventing you from getting results.

**Comment by ino89** [comment]
Ip:148.251.184.12:7786 query port 27165

**Comment by ino89** [comment]
Sry I forgot to write
Ip:148.251.184.12:7786 query port 27165

**Comment by Austinb** [comment]
The client connect port and the query port you have do not follow the normal port layout.  From what I found the query port is 27165 - 19378 which gets you a client port of 7787 which is the default connect port.  Is the query port static for this game?

For now you will need to add the server with the query port defined to get it to query the correct port.
```php
$gq->addServer([
    'type' => 'squad',
    'host' => '148.251.184.12:7786',
    'options' => [
        'query_port' => 27165,
    ],
]);
```

Try that out and let me know if it is not working for you. I ran it and I got a response from a server.
